metaflow/plugins/kubernetes/kubernetes_client.py

```diff
@@ -4,7 +4,7 @@
 
 from metaflow.exception import MetaflowException
 
-from .kubernetes_job import KubernetesJob
+from .kubernetes_job import KubernetesJob, KubernetesJobSet
```

Comment:
Need this import for Kubernetes clients which maybe getting used via extensions.